src/PowerLincModem.ts

```diff
@@ -1,3 +1,8 @@
+import logger from 'debug';
+/* Configuring logging */
+const debug = logger('node-red-contrib-insteon:powerLincModem');
```

Comment:
This references `node-red-contrib-insteon` but this is the plm library.

Reply:
Mistake. Should be `insteon-plm`.